client/my-sites/checkout/composite-checkout/composite-checkout.js

```diff
@@ -627,6 +632,31 @@ function useCountryList( overrideCountryList ) {
 	return countriesList;
 }
 
+function useRecordCheckoutLoaded(
+	recordEvent,
+	isApplePayAvailable,
+	isApplePayLoading,
+	responseCart,
+	storedCards,
+	isLoadingStoredCards,
+	product
+) {
+	const hasRecordedCheckoutLoad = useRef( false );
+	if ( ! isLoadingStoredCards && ! isApplePayLoading && ! hasRecordedCheckoutLoad.current ) {
+		debug( 'composite checkout has loaded' );
+		recordEvent( {
+			type: 'CHECKOUT_LOADED',
+			payload: {
+				saved_cards: storedCards.length,
+				apple_pay_available: isApplePayAvailable,
+				product_slug: product,
+				is_renewal: hasRenewalItem( responseCart ),
```

Comment:
Is there a chance here that `responseCart` hasn't loaded yet? I don't think that apple pay or stored cards wait on it. Should we also wait on `isLoadingCart`?